test/integration/api/backend_api_configs_controller_test.rb

```diff
@@ -17,10 +17,10 @@ class Api::BackendApiConfigsControllerTest < ActionDispatch::IntegrationTest
   attr_reader :service, :backend_api
 
   test '#index' do
-    service.backend_api_configs.create(backend_api: backend_api)
+    service.backend_api_configs.create!(backend_api: backend_api, path: 'whatever')
```

Comment:
Why does this only affect Oracle?

Reply:
@guicassolato no idea :woman_shrugging: It could be that it is not database related but just randomly regardless of the DB.

Reply:
@guicassolato 
I am running tests in all DBs in other PRs and apparently it happens for Oracle tests always and only Oracle. But that is not necessarily related to the DB itself as I discovered for https://github.com/3scale/porta/pull/1205

Reply:
Because empty String  is `nil` in oracle?
Can you investigate this clue?

Reply:
no @hallelujah , it is not related to that. The error message is `ActiveRecord::RecordNotUnique: OCIError: ORA-00001: unique constraint (RAILS.INDEX_BACKEND_API_CONFIGS_ON_PATH_AND_SERVICE_ID) violated` and it happens because in https://github.com/3scale/porta/pull/1292 we associate a backend api with empty path for every first service of a tenant, so now it tries to create a backend api with empty path again (empty path I mean '/') for the same service and it says that it can't.